### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Color Contrast: MAS1.4.3: Non-active tabs (Allocations/Call tree/Snapshots/Root & Cycles/Console)  of Allocations and Non-active tabs (Call tree/Samples/Console) of Time Profiler are not having color contrast ratio of 4.5:1

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#448]
### Pre-Requisite:
1. Install Xamarin Profiler.
2. Launch Visual Studio.
3. Create a project and build the solution.
4. Navigate to Run Menu Tab and Invoke it. Navigate to "Start Profiling" option and invoke it.
5. This will invokes "Xamarin Profiler" application.

### Steps to Reproduce

Scenario 1:
1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Allocations" and invoke next button. Invoke Start Profiling button.
3.This will invoke "Allocaitons" Profiling page.
4. Verify the color contrast ratio of non-active tabs among (Allocations/Call tree/Snapshots/Root & Cycles/Console)

Scenario 2:
1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Time Profiler" and invoke next button. Invoke Start Profiling button.
3.This will invoke "Time Profiler" Profiling page.
4. Verify the color contrast ratio of non-active tabs among (Call tree/Samples/Console)

### Expected Behavior
Color contrast ratio should be minimum 4.5:1
### Actual Behavior
Color contrast ratio is less than 4.5:1 i.e, 3.5:1
See the screenshots attached
### User Impact
Low vision user will feel difficult in seeing the text
### Screenshots
![4 tabs allocations](https://user-images.githubusercontent.com/22956130/41722711-eb182186-7586-11e8-8191-a7d884d9d9a0.png)
![5 tabs time profiler](https://user-images.githubusercontent.com/22956130/41722723-f2bf1426-7586-11e8-89bc-04b4401abec5.png)

### Environment
macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following tags to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Profiler-MAC
Wipro
